src/providers/OptimizedSubscriptionProvider.tsx

```
@@ -47,20 +51,23 @@ export function OptimizedSubscriptionProvider({ children }: OptimizedSubscriptio
   // Initialize the optimized subscription manager
   useEffect(() => {
     const manager = OptimizedSubscriptionManager.getInstance(queryClient);
-    
+
     // Optimized refresh function
     const refreshSubscriptions = useCallback(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)